zipkin-ui/js/component_data/trace.js

```diff
@@ -3,13 +3,22 @@ import $ from 'jquery';
 import {getError} from '../../js/component_ui/error';
 import traceToMustache from '../../js/component_ui/traceToMustache';
 
+function toContextualLogsUrl(logsUrl, traceId) {
+  if (logsUrl !== undefined) {
```

Comment:
if you export this function, you should be able to test it by making a class test/component_data/trace.test.js 

something like... (please use the exact format of kibana search url you use as that helps folks understand how it is used)
```javascript
import {toContextualLogsUrl} from '../../js/component_data/trace';

describe('toContextualLogsUrl', () => {
  it('replaces token in logsUrl when set', () => {
    toContextualLogsUrl('http://some-realistic-logs-url/{traceId}', 'abcd').should.equal(''http://some-realistic-logs-url/abcd'');
  });
});
```